www/artists/cfarrero/gallery.html

```diff
@@ -45,7 +45,7 @@ <h2>
                 Fiber, too: Unknown material
             </p>
             <h2>
-                These pieces are Charley Farrero collaboration work with <a href="../jjacobs/index.html"></a>June J. Jacobs.
+                These pieces are Charley Farrero collaboration work with <a href="https://saskartists.ca/artists/jjacobs/">June J. Jacobs.</a>
```

Comment:
Avoid absolute links to SaskArtists. Still, nice catch with the closing tag position.